test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/RepositorySignatureInfoUtilityTests.cs

```diff
@@ -0,0 +1,302 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using NuGet.Common;
+using NuGet.Packaging.Core;
+using NuGet.Packaging.Signing;
+using Test.Utility.Signing;
+using Xunit;
+
+namespace NuGet.Packaging.Test
+{
+    public class RepositorySignatureInfoUtilityTests
+    {
+        private static SignedPackageVerifierSettings _defaultSettings = SignedPackageVerifierSettings.GetDefault();
+        private static SignedPackageVerifierSettings _verifyCommandDefaultSettings = SignedPackageVerifierSettings.GetVerifyCommandDefaultPolicy();
+
+        [Fact]
+        public void GetSignedPackageVerifierSettings_NullCommonSettingsThrows()
+        {
+            // Arrange & Act
+            Action action = () => RepositorySignatureInfoUtility.GetSignedPackageVerifierSettings(null, null);
+
+            // Assert
+            action.ShouldThrow<ArgumentNullException>();
+        }
+
+        [Fact]
+        public void GetSignedPackageVerifierSettings_NullRepoSignatureInfoReturnsCommonSettings()
+        {
+            // Arrange
+            var commonSettings = _defaultSettings;
```

Comment:
Remove this local and just use `_defaultSettings` below.  Same with every other test.  There's no value in capturing a field in a local like this.  `SignedPackageVerifierSettings` is immutable.